.github/pull_request_template.md

```diff
@@ -0,0 +1,8 @@
+
+
+
+## Code Review Checklist
```

Comment:
Make this an actual check list. github supports:

- [ ] Alpha
- [ ] Beta
- [ ] Gamma

Reply:
syntax `- [ ] text`

Reply:
unfortunately doesn't support two checkboxes side-by-side

Reply:
oh nice